app/controllers/robot_verifications_controller.rb

```diff
@@ -32,8 +35,7 @@ def download
     @robot = Robot.find(params[:robot_id])
     @robot_verification = @robot.verification_behaviour
     if @robot_verification.valid_submission?(params)
-      @robot_verification.record_plate_types(params[:source_plate_types])
-      @robot_verification.record_plate_types(params.fetch(:destination_plate_types, {}))
+      @robot_verification.record_plate_types(params[:plate_types])
       @batch = Batch.find(params[:batch_id])
```

Comment:
As each of these is indexed by plate barcode, there is no reason they couldn't share a single parameter. This avoids the need to add control_plate_types and yet another row.